test/unit/storage_async_test.rb

```diff
@@ -0,0 +1,228 @@
+require File.expand_path(File.dirname(__FILE__) + '/../test_helper')
+
+class StorageAsyncTest < Test::Unit::TestCase
+  def test_basic_operations
+    storage = StorageAsync::Client.instance(true)
+    storage.del('foo')
+    assert_nil storage.get('foo')
+    storage.set('foo', 'bar')
+    assert_equal 'bar', storage.get('foo')
+  end
+
+  def test_redis_host_and_port
+    storage = StorageAsync::Client.send :new, url('127.0.0.1:6379')
+    assert_connection(storage)
+  end
+
+  def test_redis_url
+    storage = StorageAsync::Client.send :new, url('redis://127.0.0.1:6379/0')
+    assert_connection(storage)
+  end
+
+  def test_redis_unix
+    storage = StorageAsync::Client.send :new, url('unix:///tmp/redis_unix.6379.sock')
```

Comment:
The PR looks nice!

Shouldn't we parametrize and make these configurable?

Reply:
Do we have instructions how to setup redis locally (or in CI) to properly run these?

Reply:
> Shouldn't we parametrize and make these configurable?

What do you mean?

> Do we have instructions how to setup redis locally (or in CI) to properly run these?

There's a link to the instructions in the description. There's also a `docs` folder in this project with some info, and a `README.md` and `DEVELOPMENT.md` files that explain the make scripts to launch the project locally. The `script` folder can provide some info as well, on how are dependencies launched.